O21.Resources/Dib.fs

```diff
@@ -33,7 +33,15 @@ type Dib(dib: byte[]) =
                                              
     member _.GetPixel(x: int, y: int): RGB =
         let y = height - y - 1 // turn the image upside-down
-        if colorDepth = 4us then
+        if colorDepth = 1us then
```

Comment:
Ok, are there really fish with color depth = 1? Or is it for transparency masks?

Reply:
Yes, all transparency masks in this dll have a color depth = 1. The color palette here contains only two colors: black (0,0,0) or white (255,255,255).